frontend/src/pages/project/index.vue

```diff
@@ -22,7 +22,7 @@
             </template>
             <template v-slot:tools>
                 <div class="space-x-2 flex">
-                    <a v-if="editorAvailable" :href="project.url" target="_blank" class="ff-btn ff-btn--secondary" data-action="open-editor">
+                    <a v-if="editorAvailable && !isVisitingAdmin" :href="project.url" target="_blank" class="ff-btn ff-btn--secondary" data-action="open-editor">
```

Comment:
I was originally included to disable the button (and add a title/tooltip explaining why), but decided against this after discussing with @joepavitt.